### ISSUE TITLE
[Bug]: Not exporting hit dice

### ISSUE THREAD

**Issue by ejwright** [bushvin/actor-export#11]
### Foundry VTT version

12.324

### Game system

dnd5e

### Game system version

3.2.0

### actor-export module version

2024.5.8

### What happened?

Hit Dice total shows as : '[object .' -- I was able to patch the provider file with

```
// Hit Dice
ref = 'hit dice';
const hitDice_size = Array.from(character.hd.sizes);
const hitDice = character.hd.max + 'd' + hitDice_size[0];
mapper.textBox(ref, fileNames, 0, 249, 313, 48, 14, hitDice, mf_12);
```

### Sample Actor

```JSON
Sorry.. doesn't let me paste it (comment too long error)
```

**Comment by bushvin** [comment]
Thank you for this. I will see to update this asap.

**Comment by bushvin** [comment]
This should be fixed with release 2024.6.2
